src/syntax/FStar.Syntax.Resugar.fs

```diff
@@ -98,46 +102,45 @@ let rec resugar_universe (u:S.universe) r: A.term =
 
 let string_to_op s =
   let name_of_op = function
-    | "Amp" -> Some "&"
-    | "At" -> Some "@"
-    | "Plus" -> Some "+"
-    (* TODO : there is some arity information which is lost here ! *)
-    | "Minus" -> Some "-"
-    | "Subtraction" -> Some "-"
-    | "Slash" -> Some "/"
-    | "Less" -> Some "<"
-    | "Equals" -> Some "="
-    | "Greater" -> Some ">"
-    | "Underscore" -> Some "_"
-    | "Bar" -> Some "|"
-    | "Bang" -> Some "!"
-    | "Hat" -> Some "^"
-    | "Percent" -> Some "%"
-    | "Star" -> Some "*"
-    | "Question" -> Some "?"
-    | "Colon" -> Some ":"
+    | "Amp" -> Some ("&", 0)
+    | "At" -> Some ("@", 0)
+    | "Plus" -> Some ("+", 0)
```

Comment:
is the second component really the arity? The comment seems to suggest so. But then, arity 0 seems wrong here. Not sure